src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -43,13 +42,13 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
      *
      * @throws ParseException if the given {@code name} is invalid.
      */
-    public static Name parseName(String name) throws ParseException {
+    public static seedu.address.model.data.Name parseName(String name) throws ParseException {
         requireNonNull(name);
         String trimmedName = name.trim();
-        if (!Name.isValidName(trimmedName)) {
-            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
+        if (!seedu.address.model.data.Name.isValidName(trimmedName)) {
+            throw new ParseException(seedu.address.model.data.Name.MESSAGE_CONSTRAINTS);
         }
-        return new Name(trimmedName);
+        return new seedu.address.model.data.Name(trimmedName);
```

Comment:
Does not look like it should be seedu.address.model.data.Name

Try importing the file?